_drafts/why-code.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Why learn to code?"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, genomics, workflows
+  - r
+  - python
+  - 
+---
+Occassionally, after someone takes an Intro R or Python course through FredHutch.io we will get feedback that boils down to the question "How am I supposed to use this in my research?". It's an understandable question - an intro course generally covers the basics of importing, exporting, subsetting, and visualizing data. This basic skillset does not clearly translate to the incredible and complex work being done in the clinics and wet-labs around campus. In this post we will go over how learning to program and understanding code could improve biology research.
```

Comment:
Let's reframe this: "so you're thinking about taking a coding course, but aren't sure if it will apply to your own research."

Reply:
Ok reworked it a bit - let me know what you think